components/Alert/alert.css

```diff
@@ -0,0 +1,21 @@
+.alert {
+  display: inline-block;
+  width: 100%;
+  padding: 0.5rem;
+  border-radius: 5px;
+  margin: 0.5rem;
```

Comment:
Never mind the selected lines of code , this comment is design review 
1.buttons are square, rest elements have rounded corners [maintain design consistency]
2.try to maintain const spaces between elements in all pages 
3.the side bar should hold the state while we are in a page (like when we press avatar page , the button color should not go away until we move to the next page).
4. make your logo bigger and bolder (it should get some focus over other elements right)